plugins/outputs/nebius_cloud_monitoring/nebius_cloud_monitoring.go

```diff
@@ -242,3 +242,13 @@ func init() {
 		return &NebiusCloudMonitoring{}
 	})
 }
+
+func replaceReservedTagNames(tagNames map[string]string) map[string]string {
+	for tagName, tagValue := range tagNames {
+		if tagName == "name" {
+			tagNames["label_name"] = tagValue
```

Comment:
can you get a screenshot or let us know how this will appear in the nebius UI? I want to be clear as to what this might look like and ensure the double "label" name makes sense.

Reply:
I agree, maybe `_name` would be better?